sources/web/datalab/polymer/components/item-list/item-list.html

```diff
@@ -182,7 +201,7 @@
           </div>
           <template is="dom-repeat" items="{{row.columns}}" as="column">
             <div class="column">
-              <span>{{column}}</span>
+              <span>[[_formatColumnValue(column, index, columns)]]</span>
```

Comment:
You are inside a nested dom-repeat here. I think it would add clarity and robustness to be explicit about which index you are using by adding index-as to the columns dom-repeat.